README.md

```diff
@@ -43,6 +49,20 @@ to the installation directory of the JDK (not to be mistaken with JRE).
 You can test if it works by calling `mvn -version` in a Terminal.
 This will provide the information about the JDK Maven is using, in case
 it is the wrong one.
+
+## Rust 1.28 (and Cargo)
```

Comment:
I wondered several times whether it would be better to have separate install instructions for each backend